Dockerfile

```diff
@@ -1,23 +1,21 @@
-FROM python:3.5-slim-stretch
+FROM python:3.7-slim-stretch
 
 RUN apt-get update && apt-get install -y \
     g++ \
     gfortran \
     libblas-dev \
     liblapack-dev \
     libopenblas-dev \
-    python3-dev \
     enchant \
     build-essential
+COPY requirements.txt /ores/requirements.txt
```

Comment:
Requirements tend to change less often than the source code. If we `COPY` the requirements before the source code, then we can speed up build time by avoiding re-installing all the dependencies.